packages/x-components/src/x-modules/tagging/service/session.service.ts

```diff
@@ -8,24 +8,41 @@ import { SessionService } from './types';
  * @public
  */
 export class DefaultSessionService implements SessionService {
-  protected readonly SESSION_ID_KEY = 'session-id';
+  /**
+   * Session id key to use as key in the storage.
+   *
+   * @public
+   */
+  public readonly SESSION_ID_KEY = 'session-id';
```

Comment:
Shouldn't this be a static property?

Reply:
I is exposed for testing purposes, but yes it can be static.